srcpkgs/black/template

```diff
@@ -1,19 +1,38 @@
 # Template file for 'black'
 pkgname=black
-version=20.8b1
+version=21.8b0
 revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools_scm"
-depends="python3-appdirs python3-click python3-mypy_extensions
- python3-pathspec python3-toml python3-regex python3-typed-ast
+hostmakedepends="python3-setuptools python3-setuptools_scm"
+depends="python3-click python3-platformdirs python3-tomli
+ python3-mypy_extensions python3-pathspec python3-regex
  python3-typing_extensions"
+checkdepends="python3-typed-ast python3-aiohttp python3-aiohttp-cors
+ python3-pytest python3-parameterized python3-ipython python3-tokenize-rt
+ $depends"
 short_desc="Uncompromising Python code formatter"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
-homepage="https://github.com/psf/black"
+homepage="https://pypi.org/project/black"
+changelog="https://github.com/psf/black/blob/main/CHANGES.md"
 distfiles="${PYPI_SITE}/b/black/black-${version}.tar.gz"
-checksum=1c02557aa099101b9d21496f8a914e9ed2222ef70336404eeeac8edba836fbea
+checksum=570608d28aa3af1792b98c4a337dbac6367877b47b12b88ab42095cfc1a627c2
+
+pre_build() {
+	# <https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-python/black/black-21.8_beta0.ebuild#n45>.
+	sed -i '/setuptools_scm/s:~=:>=:' setup.cfg
```

Comment:
Can you use vsed here? vsed is sed that prints a warning when it doesn't change anything.